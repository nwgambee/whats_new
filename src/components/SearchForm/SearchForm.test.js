import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import Form from './SearchForm';

describe('Form', () => {
  it('should match the snapshot', () => {
    const wrapper = shallow(<Form
      onClick={jest.fn()}
      value='some query'
      onChange={jest.fn()}
         />);

    expect(wrapper).toMatchSnapshot();
  });
});
