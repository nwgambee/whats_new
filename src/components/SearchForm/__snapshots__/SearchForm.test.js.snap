// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form should match the snapshot 1`] = `
<form>
  <input
    name="title"
    onChange={[Function]}
    placeholder="Title..."
    type="text"
    value=""
  />
  <button
    className="submit-btn"
    name="submit-btn"
    onClick={[Function]}
    type="submit"
  >
    Submit Idea
  </button>
</form>
`;
